build-tests-samples/heft-serverless-stack-tutorial/tsconfig.json

```diff
@@ -22,7 +22,7 @@
 
     "module": "commonjs",
     "target": "es2017",
-    "lib": ["es2017"]
+    "lib": ["es2017", "DOM"]
```

Comment:
Related?

Reply:
Yes, `esbuild` needs the typings for the `WebAssembly` namespace, which is provided by the `DOM` lib in TypeScript.